### ISSUE TITLE
default directory after a program is launched?

### ISSUE THREAD

**Issue by bsmile001** [cs01/gdbgui#240]
**Describe the bug**
I just start using gdbgui, so my questions might not be issues at all. But I do need help on them.  One thing is I cannot click to choose the target executable but have to paste its path, is it the case?  Second, after gdbgui loads the executable, shell pwd still reports the default directory to be home directory. Does the directory coincide with where gdbgui is launched? I hope we could be given the freedom to choose the working directory where input files resides.

**To Reproduce**

**Expected behavior**
(1) give windows like click to choose directories for exectuable
(2) provide way to specify working directory.

**Screenshots**

**Please complete the following information:**
* OS: linux mint 18.2 installed in a virtualbox with windows 7 as host
* gdbgui version (`gdbgui -v`):0.13.0.0
* gdb version (`gdb -v`):7.11.1
* browser [e.g. chrome, safari]: firefox 62.0.3 in linux
* python packages (`pip freeze`): pip 8.1.1

**Additional context**

**Comment by cs01** [comment]
> One thing is I cannot click to choose the target executable but have to paste its path, is it the case?

Yes. There is no file explorer in the gui. I hope to add one someday, but it's not high on the priority list. You can also tell gdbgui which executable to use when launching, `gdbgui <executable>`.

> Second, after gdbgui loads the executable, shell pwd still reports the default directory to be home directory. Does the directory coincide with where gdbgui is launched?

Yes. You can change it in the terminal at the bottom of the browser using standard shell commands `cd` and `pwd`.

**Comment by bsmile001** [comment]
> 
> 
> > One thing is I cannot click to choose the target executable but have to paste its path, is it the case?
> 
> Yes. There is no file explorer in the gui. I hope to add one someday, but it's not high on the priority list. You can also tell gdbgui which executable to use when launching, `gdbgui <executable>`.

It seems gdbgui <executable> take a long time still not finishing initializing. I am using linux mint 18.2 in a virtualbox with win 7 as host. I thus use gdbgui and open executable within gdbgui, which is much smoother.
> 
> > Second, after gdbgui loads the executable, shell pwd still reports the default directory to be home directory. Does the directory coincide with where gdbgui is launched?

Thanks! This way works for me.
